tools/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/moby/moby/tools
+
+go 1.17
+
+require github.com/containerd/protobuild v0.1.1-0.20211025221430-7e5ee24bc1f7
+
+require (
+	github.com/golang/protobuf v1.5.0 // indirect
```

Comment:
Should we not use the same version as in our `vendor.mod`?

Reply:
Ah, yes, good catch 😅

Reply:
Done!